Build/Shared.Extensions.targets

```diff
@@ -61,21 +61,40 @@
       <AllDatabaseSystems Include="SqlServer2016" Condition="'%(DatabaseSystemsList.Identity)' == 'SqlServer2016'"/>
       <AllDatabaseSystems Include="SqlServer2017" Condition="'%(DatabaseSystemsList.Identity)' == 'SqlServer2017'"/>
     </ItemGroup>
+
+    <ItemGroup>
+      <AllBrowsers Remove="@(AllBrowsers)"/>
+
+      <AllBrowsers Include="Chrome" Condition="'%(BrowsersList.Identity)' == 'Chrome'" />
+      <AllBrowsers Include="Edge" Condition="'%(BrowsersList.Identity)' == 'Edge'" />
+      <AllBrowsers Include="InternetExplorer" Condition="'%(BrowsersList.Identity)' == 'InternetExplorer'" />
+    </ItemGroup>
   </Target>
 
   <Target Name="After_BuildTestOutputFilesCrossProduct" AfterTargets="BuildTestOutputFilesCrossProduct">
     <ItemGroup>
       <_testOutputFilesWithInternetExplorer Include="@(_testOutputFiles)" Condition="'%(Browser)' == 'InternetExplorer'" />
+      <_testOutputFilesWithEdge Include="@(_testOutputFiles)" Condition="'%(Browser)' == 'Edge'" />
     </ItemGroup>
 
     <ItemGroup Condition="'@(_testOutputFilesWithInternetExplorer->Count())' &gt; 1">
-      <_testOutputFilesWithIEFiltered Include="@(_testOutputFiles)" Condition="'%(Browser)' != 'InternetExplorer'" />
-      <_testOutputFilesWithIEFiltered Include="@(_testOutputFiles)" Condition="'%(Browser)' == 'InternetExplorer' And '%(Platform)' == 'x86' And '$(ConfigurationID)' == 'Debug'"/>
+      <_testOutputFilesWithInternetExplorerReduced Include="@(_testOutputFiles)" Condition="'%(Browser)' != 'InternetExplorer'" />
+      <_testOutputFilesWithInternetExplorerReduced Include="@(_testOutputFiles)" Condition="'%(Browser)' == 'InternetExplorer' And '%(Platform)' == 'x86' And '$(ConfigurationID)' == 'Debug'"/>
     </ItemGroup>
 
     <ItemGroup Condition="'@(_testOutputFilesWithInternetExplorer->Count())' &gt; 1">
       <_testOutputFiles Remove="@(_testOutputFiles)" />
-      <_testOutputFiles Include="@(_testOutputFilesWithIEFiltered)"/>
+      <_testOutputFiles Include="@(_testOutputFilesWithInternetExplorerReduced)"/>
+    </ItemGroup>
+
+    <ItemGroup Condition="'@(_testOutputFilesWithEdge->Count())' &gt; 1">
+      <_testOutputFilesWithEdgeReduced Include="@(_testOutputFiles)" Condition="'%(Browser)' != 'Edge'" />
+      <_testOutputFilesWithEdgeReduced Include="@(_testOutputFiles)" Condition="'%(Browser)' == 'Edge' And '%(Platform)' == 'x86' And '$(ConfigurationID)' == 'Debug'"/>

```

Comment:
Why are we running Edge on the same configuration as IE?
Why are we not removing Chrome for the run that is using Edge (and IE, actually - new thing)?